CONTRIBUTING.org

```diff
@@ -0,0 +1,186 @@
+* Contributing a new recipe to MELPA
+
+MELPA consists of /recipes/, each describing one /package/ in a
+dedicated repository. This document describes how to propose a new
+package for inclusion into MELPA.
+
+Three contributor roles are involved in the maintenance of MELPA :
+
+- MELPA maintainers :: They are responsible for reviewing new recipes
+     and associated packages as well as keeping the system working.
+
+- Recipe authors :: They responsible for submitting good recipes and
+                    maintaining them when the package is changed
+                    (e.g., renamed or deprecated).
+
+- Package author :: They are responsible for writing Emacs Lisp
+                    packages of good quality.
+
+We advise package author to also take the recipe author role when
+possible.
+
+** What to know before opening a pull request
+
+New recipe submissions should adhere to the following guidelines:
+
+- One recipe per pull request :: If you have several packages to
+     submit, open several pull requests (even if the packages depend
+     on each other).
+
+- Reasonably innovative package :: MELPA provides a curated set of
+     Emacs Lisp packages, not an exhaustive list of every single Emacs
+     Lisp file ever created. As such, MELPA maintainers tend to only
+     accept packages not duplicating existing packages. Please try to
+     improve existing packages instead of creating new ones when
+     possible.
+
+- Software Configuration Management (SCM) :: Upstream source must be
+     stored in an authoritative [[https://en.wikipedia.org/wiki/Software_configuration_management][SCM repository]]. [[https://www.emacswiki.org/][EmacsWiki]] recipes are
+     no longer accepted.
+
+- Official repository :: Packages should be built from the official
+     package repository.  Forks of the official repository will not be
+     accepted except in extreme circumstances.
+
+- Dedicated SCM repository :: Keep each package in its own SCM (e.g.,
+     git) repository. This makes it possible to have a different
+     version number for each package as MELPA stable looks at the SCM'
+     tags to assign version numbers to recipes.
```

Comment:
In a separate item you should mention how to make a package independent of a certain completion framework, by using `dont-remember-ask-steve`.

You should place that item after this one, because it is an exception to the one-package-per-repo rule: when splitting e.g. `helm-foo` from `foo`, it is perfectly okay (though not required) to keep both libraries in the same repository.